### PR TITLE
Update container file

### PR DESCRIPTION
**Pull request by elfein727**

### CODE REVIEW

File: containers/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,8 @@
+# Base image can be found here: https://hub.docker.com/r/jrottenberg/ffmpeg/
+# There are options for selecting a different distro.
+FROM jrottenberg/ffmpeg:3.3
+LABEL Name=horny_jail Version=0.0.1
+ADD https://github.com/yt-dlp/yt-dlp/releases/latest/download/yt-dlp /bin/
```
Comment (StellaSmith) [inline containers/Dockerfile:5]: maybe copy it to `/usr/local/bin`?
also, im not quite sure if this sets it as executable

File: containers/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,8 @@
+# Base image can be found here: https://hub.docker.com/r/jrottenberg/ffmpeg/
+# There are options for selecting a different distro.
+FROM jrottenberg/ffmpeg:3.3
```
Comment (StellaSmith) [inline containers/Dockerfile:3]: use the full name: `docker.io/jrottenberg/ffmpeg:3.3-ubuntu`
also, why not `4.1-vaapi`?

File: containers/Dockerfile [hunk 3]
```diff
@@ -0,0 +1,8 @@
+# Base image can be found here: https://hub.docker.com/r/jrottenberg/ffmpeg/
+# There are options for selecting a different distro.
+FROM jrottenberg/ffmpeg:3.3
+LABEL Name=horny_jail Version=0.0.1
+ADD https://github.com/yt-dlp/yt-dlp/releases/latest/download/yt-dlp /bin/
+WORKDIR /home/root/
+COPY /{path to binary} ./
```
Comment (StellaSmith) [inline containers/Dockerfile:7]: the easiest would be to have everything under `/opt/pervybot/`

**Review by StellaSmith — CHANGES_REQUESTED** [review]

File: containers/Dockerfile (hunk 1 above)
Comment (StellaSmith) [inline containers/Dockerfile:5]: Also, this requires python, which is not available in this image container

**Review by StellaSmith — CHANGES_REQUESTED** [review]

File: containers/Dockerfile (hunk 1 above)
Comment (elfein727) [reply containers/Dockerfile:5]: Using new image now with the binary included.

File: containers/Dockerfile (hunk 2 above)
Comment (elfein727) [reply containers/Dockerfile:3]: Using `docker.io/tnk4on/yt-dlp` now, which might not have this, but it exposes ffmpeg and yt-dlp.

File: containers/Dockerfile (hunk 3 above)
Comment (elfein727) [reply containers/Dockerfile:7]: Done.

File: containers/Dockerfile (hunk 1 above)
Comment (elfein727) [reply containers/Dockerfile:5]: Solved by switching images.

**Review by StellaSmith — CHANGES_REQUESTED** [review]
You can build the binary inside the Dockerfile, and copy it to the final image.

See https://youtu.be/uYhLWN86V48?t=486 as an example

File: Dockerfile [hunk 4]
```diff
@@ -0,0 +1,13 @@
+FROM rust:latest as build
```
Comment (StellaSmith) [inline Dockerfile:1]: Use the full name: docker.io/library/rust:1-slim

**Review by StellaSmith — CHANGES_REQUESTED** [review]

File: Dockerfile [hunk 5]
```diff
@@ -0,0 +1,13 @@
+FROM rust:latest as build
+# RUN rustup target add x86_64-unknown-linux-musl
+WORKDIR /usr/src/pervybot-rs
+COPY . .
+RUN cargo build --release
+
+# https://hub.docker.com/r/tnk4on/yt-dlp
+FROM docker.io/tnk4on/yt-dlp
+LABEL Name=pervy_jail Version=0.0.1
```
Comment (StellaSmith) [inline Dockerfile:9]:
```suggestion
FROM docker.io/tnk4on/yt-dlp
USER root:root
```

**Review by StellaSmith — CHANGES_REQUESTED** [review]

File: .dockerignore [hunk 6]
```diff
@@ -0,0 +1,2 @@
+target
+target/*
```
Comment (StellaSmith) [inline .dockerignore:2]:
```suggestion
target/
```

**Review by StellaSmith — APPROVED** [review]

**Pull request merged**